roles/katello_test/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: 'Loosen pulpcore gem restrictions'
+  ansible.builtin.lineinfile:
+    path: /home/vagrant/katello/katello.gemspec
+    regexp: 'gem.add_dependency "{{ item }}"'
+    line: '  gem.add_dependency "{{ item }}"'
+  loop:
+    - "pulpcore_client"
+    - "pulp_file_client"
+    - "pulp_ansible_client"
+    - "pulp_container_client"
+    - "pulp_deb_client"
+    - "pulp_rpm_client"
+    - "pulp_2to3_migration_client"
+    - "pulp_certguard_client"
+
+- name: 'env'
+  ansible.builtin.shell: |
+    id
+  args:
+    executable: /bin/bash
+    chdir: "{{ foreman_development_dir }}"
+  become_user: "{{ test_user }}"
```

Comment:
What is this step doing? Looks like its just printing out user information?

Reply:
That's correct - not needed and was just for debugging purposes.